packages/devtools-network-console/src/host/web-application-host.ts

```diff
@@ -46,6 +46,9 @@ export default class WebApplicationHost implements INetConsoleHost {
         if (request.url === 'wss://www.norad.mil/cheyenne/WOPR') {
             WebSocketMock.instance('wss');
             const time = Math.random() * 1000;
+            setTimeout(() => {
+                globalDispatch(makeWebSocketConnectedAction('DEFAULT_REQUEST'));
+            }, Math.max(time, 0));
```

Comment:
The expression `Math.max(time, 0)` in this case will always equal `time`.

Reply:
oops. I swear it made sense when I wrote it.... :D 

The goal is to make sure that this gets sent before the first mock message, but a bit after the "request_complete" message. I've updated to send on time/2